targets/TARGET_RENESAS/TARGET_VK_RZ_A1H/gpio_irq_api.c

```diff
@@ -71,6 +71,7 @@ static const PinMap PinMap_IRQ[] = {
 #else
 static const PinMap PinMap_IRQ[] = {
     {P9_1,  IRQ0, 4},
+    {P7_8,  IRQ1, 8},
```

Comment:
Could you move it into another PR? We can get both in faster if they are smaller and more targeted.